swagger.yaml

```diff
@@ -91,6 +96,122 @@ paths:
           $ref: "#/responses/InternalError"
         502:
           $ref: '#/responses/BadGateway'
+  /population-types/{population-type}/dimensions:
+    get:
+      summary: "Get list of non-geography base variables for given dataset "
+      description: ""
+      produces:
+        - "application/json"
+      parameters:
+        - name: population-type
+          type: string
+          in: path
+          required: true
+      responses:
+        200:
+          description: "A json list containing non-geography base variables for a dataset"
+          schema:
+            $ref: '#/definitions/Dimension'
+        401:
+          $ref: '#/responses/Unauthorised'
+        404:
+          description: "Dataset not found"
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          $ref: "#/responses/InternalError"
+        502:
+          $ref: '#/responses/BadGateway'
+  /population-types/{population-type}/dimensions/{dimension}/categorisations:
+    get:
+      summary: "Get list of categorisations for a base level variable"
+      description: ""
+      produces:
+        - "application/json"
+      parameters:
+        - name: population-type
+          type: string
+          in: path
+          required: true
+        - name: dimension
+          type: string
+          in: path
+          required: true
+      responses:
+        200:
+          description: "A json list containing categorisations for a base variable"
+          schema:
+            $ref: '#/definitions/Dimension'
+        401:
+          $ref: '#/responses/Unauthorised'
+        404:
+          description: "Dataset not found"
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          $ref: "#/responses/InternalError"
+        502:
+          $ref: '#/responses/BadGateway'
+  /population-types/{population-type}/dimensions/{dimension}/base:
+    get:
+      summary: "Get the base variable for a categorisation"
+      description: ""
+      produces:
+        - "application/json"
+      parameters:
+        - name: population-type
+          type: string
+          in: path
+          required: true
+        - name: dimension
+          type: string
+          in: path
+          required: true
+      responses:
+        200:
+          description: "The base variable information"
+          schema:
+            $ref: '#/definitions/Base'
+        401:
+          $ref: '#/responses/Unauthorised'
+        404:
+          description: "Dataset not found"
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          $ref: "#/responses/InternalError"
+        502:
+          $ref: '#/responses/BadGateway'
+  /population-types/{population-type}/dimensions/{dimension}/parents:
```

Comment:
I can't find a handler for this endpoint. Is this in anticipation of future work?

Reply:
This is no longer needed so has been removed